applovin_max/lib/applovin_max.dart

```diff
@@ -43,6 +43,12 @@ class AppLovinMAX {
   /// See [this GitHub issue](https://github.com/AppLovin/AppLovin-MAX-Flutter/issues/210)
   /// for more details on why calling `initialize` multiple times can lead to issues.
   static Future<MaxConfiguration?> initialize(String sdkKey) async {
+    bool? isPlatformSDKInitialized = await isInitialized();
```

Comment:
naming nit: `isInitialized` (I don't even know if convention is to capitalize `SDK` or `Sdk`.

Reply:
Can we nil coalesce here instead? It's easier to read the code and you won't need optional.

```suggestion
    bool isInitialized = await isInitialized() ?? false;
```

Reply:
If pub calls `initialize(...)` two times one line after another asynchronously, do we return two `Future`s?

Reply:
Currently, the 2nd `initialize()` won't return.  Added `Completer` to return a future object for the 2nd `Future`.